recipes/guidata/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "1.7.6" %}
+
+package:
+  name: guidata
+  version: {{ version }}
+
+source:
+  url: https://github.com/PierreRaybaut/guidata/archive/v{{ version }}.tar.gz
+  sha256: 2ffc8e6dd27a57c6ec0e260986368beeddbd2fa5588f5200614b307b27a39e37
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```

Comment:
@marcelotrevisani the [entrypoint seems to be versioned](https://github.com/PierreRaybaut/guidata/blob/master/setup.py#L149) so we cannot use `noarch` here. Can you add the entry point here and a test below? Here is an example:

https://github.com/conda-forge/ipython-feedstock/blob/master/recipe/meta.yaml#L16-L18

Reply:
Sure! I will see how to do that and I will try it! Thanks!

Reply:
Just one question, do I have to patch the ```setup.py``` and remove the entry point there?

Reply:
No need to patch, although versioned entrypoints is a bad practice IMO :smile: 

You need to add two entrypoints lines for the conda-recipe with the proper selector, something like:

```yaml
  entry_points:
    - guidata-tests-py2 = guidata.tests:run  # [py2k]
    - guidata-tests-py3 = guidata.tests:run  # [py3k]
```


(Untested!)

PS: I'm not a `guidata` user and I don't really know if these entrypoints are useful, if they are not you can patch to remove them altogether and package this as `noarch`.

Reply:
Idk why appveyor is failing

Reply:
You'll need to remove `noarch` like I did in the previous commit. We cannot use `noarch` with versioned entrypoints anyway.

Reply:
I already removed! thanks! Do I have to squash the commits?